easybuild/easyconfigs/s/s3fs/s3fs-2024.9.0-foss-2024a.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = "PythonBundle"
+
+name = 's3fs'
+version = '2024.9.0'
+
+homepage = 'https://github.com/fsspec/s3fs/'
+description = """S3FS builds on aiobotocore to provide a convenient Python filesystem interface for S3.."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('Python-bundle-PyPI', '2024.06'),
+    ('hatchling', '1.24.2'),
+    ('aiohttp', '3.10.10'),
+    ('wrapt', '1.16.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('jmespath', '1.0.1', {
+        'checksums': ['90261b206d6defd58fdd5e85f478bf633a2901798906be2ad389150c5c60edbe'],
+    }),
+    ('botocore', '1.35.36', {
+        'checksums': ['354ec1b766f0029b5d6ff0c45d1a0f9e5007b7d2f3ec89bcdd755b208c5bc797'],
+    }),
+    ('aioitertools', '0.11.0', {
+        'checksums': ['42c68b8dd3a69c2bf7f2233bf7df4bb58b557bca5252ac02ed5187bbc67d6831'],
+    }),
+    ('aiobotocore', '2.15.2', {
+        'checksums': ['9ac1cfcaccccc80602968174aa032bf978abe36bd4e55e6781d6500909af1375'],
+    }),
```

Comment:
maybe we move aiobotocore to boto3, drop it from here as well as dropping jmespath, botocore  and just depend on boto3 instead?

also aioitertools 0.12.0

Reply:
I removed unneeded deps, this should be good to go